api/python/provisioner/commands/upgrade/set_swupgrade_repo.py

```diff
@@ -652,18 +652,27 @@ def _pre_repo_validation(self, params: inputs.SWUpgradeRepo,
             Raise this exception if candidate repository validation fails
 
         """
-        if not params.is_remote():
-            if params.sig_file:
-                self._check_iso_authenticity(params)
-            elif params.hash:
+        if not dry_run and not params.is_remote():
```

Comment:
please activate that for dry run as well

Reply:
Done